test/unit_tests/modules/util/breakpoint-state-spec.js

```diff
@@ -40,7 +40,7 @@ describe( 'getBreakpointState', () => {
     let trueValueCount = 0;
 
     breakpointState = getBreakpointState();
-    for ( let stateKey in breakpointState ) { // eslint-disable-line guard-for-in, no-inline-comments, max-len
+    for ( const stateKey in breakpointState ) { // eslint-disable-line guard-for-in, no-inline-comments, max-len
```

Comment:
Interesting, this works? I guess so, so it defines a new constant every loop. I guess that makes sense.

Reply:
Yep, that seems to be the case.